packages/quick_actions/quick_actions/test/quick_actions_test.dart

```diff
@@ -17,15 +17,15 @@ void main() {
     });
 
```

Comment:
It would be good to still have a very simple test which will test the constructor in isolation. Something like the following would do the trick:

```dart
test('ctor() should return valid QuickActions instance', () {
  const QuickActions quickActions = QuickActions();
  expect(quickActions, isNotNull);
});
```